docs/SRS/SRS.tex

```diff
@@ -113,9 +129,91 @@ \subsection{Undesired Scenario Handling}
 
 \section{System Level Variables}
 \subsection{Constants}
+
+\begin{table}[H]
+  \centering
+      \setlength{\leftmargini}{0.4cm}
+      \begin{tabular}{| >{\centering\arraybackslash}m{2cm} | 
+        >{\centering\arraybackslash}m{2cm} | 
+        >{\centering\arraybackslash}m{10cm} |}
+      \hline
+      \rowcolor[gray]{0.9}
+      Constant & Unit & Value\\
+      \hline
+      Chess board width & inches & 12\\
```

Comment:
Should we chose a double the board size just so we have more space for wiring and other components to go through? 1foot by 1foot seems too small